components/Volunteer/VolunteerOpportunityList.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { VolunteerOpportunity } from 'config/types'
+import { VolunteerOpportunityDetail } from './VolunteerOpportunityDetail'
+import { StyledList } from './VolunteerOpportunityDetail.styled'
+
+interface VolunteerOpportunityListProps {
+  opportunities: VolunteerOpportunity[]
+}
+
+export const VolunteerOpportunityList = ({ opportunities }: VolunteerOpportunityListProps) => (
+  <StyledList>
+    {opportunities.map((opportunity, i) => {
+      return <li><VolunteerOpportunityDetail key={i} opportunity={opportunity} /></li>
```

Comment:
There's a warning in the console about a missing key. I think you'll need to move it from the `<VolunteerOpportunityDetail>` onto the `<li>`

Reply:
@bplowry Hmm, where's that warning appearing? I haven't seen it... I'll shift it across nonetheless.

Reply:
In the browser console on the volunteering page